src/main/java/no/nav/fo/veilarbregistrering/db/registrering/AktiveringTilstandRepositoryImpl.java

```diff
@@ -110,6 +110,15 @@ public int hentAntall(Status status) {
         return antall;
     }
 
+    @Override
+    public Optional<AktiveringTilstand> hentTilstandFor(long registreringsId) {
+        String sql = "SELECT * FROM REGISTRERING_TILSTAND" +
+                " WHERE BRUKER_REGISTRERING_ID = ?" +
```

Comment:
Jeg ser vi ikke har noen unik-begrensning på denne FKen mot `BRUKER_REGISTRERING.ID`, som betyr at vi kan i teorien ha flere `REGISTRERING_TILSTAND`-rader for samme registrering. I det tilfellet vil vi her kun få én av radene, som kan forårsake merkelige tilstander/feil.

Ut ifra koden, så skal vi «aldri» få en tilstand uten en registrering ettersom begge opprettes sammen i samme transaksjon, men vil det alltid være sånn? Burde vi i så fall legge til en unik-sjekk på `REGISTRERING_TILSTAND.BRUKER_REGISTRERING_ID`?

Reply:
Godt poeng - opprinnelig tenkte vi litt på å ha historikk, men enn så lenge har vi ikke det, og da burde det kanskje være en sikkerhetsmekanisme her. Det vil sikre oss mod odde feil, så typisk tar lang tid å forstå (fordi feilen oppstår lenge etter at rotårsaken ble opprettet)

Reply:
Enig. Bedre å være strict nå og heller løse opp senere hvis behovet for det dukker opp.